Formula/kvrocks.rb

```diff
@@ -0,0 +1,28 @@
+class Kvrocks < Formula
+  desc "Key-value NoSQL database based on RocksDB and compatible with Redis protocol"
+  homepage "https://github.com/bitleak/kvrocks/"
+  url "https://github.com/bitleak/kvrocks.git",
+    tag:      "v2.0.1",
+    revision: "5f41145adb25a79706a68d233a28a0f633ef255f"
```

Comment:
Can we use a tarball instead of a Git clone?

Reply:
No actually, we need the .git directory to fetch submodules

Reply:
Are there submodules needed outside of the `external` directory? If the only submodules needed are the ones in `external`, then we definitely want a tarball instead of a Git clone.

Reply:
Looking more closely, it seems we just need submodules for the `external` directory. This means that we _need_ a tarball rather than a Git clone, since we don't want to use the `external` directory.

Reply:
yeah, we can use the tarball if changes to Homebrew's versions of those dependencies